.config/dotnet-tools.json

```diff
@@ -2,11 +2,5 @@
   "version": 1,
   "isRoot": true,
   "tools": {
-    "dotnet-mergenupkg": {
```

Comment:
not needed any more?

Reply:
no, is not not needed anymore.
before with `dotnet mergenupkg` i added the .NET tools (from the `netcoreapp` package) inside the package with `tools/paket.exe`.
now it does the opposite, it create the .NET tools package (netcoreapp) with `dotnet pack` and when packaging add also the already built `tools/paket.exe`.

Now is easier to do with .net sdk but it also needed some tweaks to build script that i added in this PR, before was just easier `dotnet mergenupkg` , to make it the build script less complicated